composer/devices/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 MosaicML Composer authors
```

Comment:
Can/should this just be a flat `composer/devices.py` file instead of a full blown module?

Reply:
Might be worth it, though punting to different PR / discussion